src/components/ConcertCard.js

```diff
@@ -43,7 +43,7 @@ const Wrapper = styled.section`
   border-radius: 10px;
   background: var(--white);
   box-shadow: var(--light-grey) 0px 2px 5px -1px, var(--black) 0px 1px 3px -1px;
-  font-family: Helvetica Neue;
+  font-family: 'Helvetica Neue';
```

Comment:
Do you need to set it here as well? You set it in your global styles, so maybe a `font-family: inherit` would be better. Or try to just remove it because it should be there.